pkg/security/secl/rules/policy.go

```diff
@@ -246,6 +246,11 @@ func (p *Policy) GetAcceptedRules() []*PolicyRule {
 	return acceptedRules
 }
 
+// GetRules returns the list of filtered rules that are part of the policy
+func (p *Policy) GetRules() map[string][]*PolicyRule {
```

Comment:
What is going to be the usage of this ? Since we return the map directly, why not make the field public ?

Reply:
Done